README.md

```diff
@@ -2,17 +2,27 @@
 
 ## Objectives
 
-  1. Return a string from a route
-  2. Use that string and replace the inner html of a div
+  1. Return a string from a route.
+  2. Use that string and replace the inner HTML of a div.
 
-## Outline
-  
-  * In here is a simple Order/Invoice/Customer application
-  * Add a product description, also an inventory to the product's Model. Create an index page of
-    products
-  * For each product make a "more info" button. 
-  * Create a new route that would give the description. Another route to check
-    if inventory is still avialable. Should just return `true` or `false`.
-  * when the user clicks more info it should return the description and if
-    there is available invetory should say so. Or sold out say not
+## Introduction
 
+Included in the lab is a simple Order/Invoice/Customer application.
+We're going to use what we've learned so far to add API endpoints and
+use AJAX to bring data to the page.
+
+## Instructions
+
+1. Add a product `description` and an `inventory` count to the `Product`
```

Comment:
This objective is a bit hard to follow. Maybe split it out and renumber accordingly? Something like

"""
1. Ad a product `description` and an `inventory` count to the `Product` model.
2. Create an `index` page for all products that displays each product's `name` and its truncated `description`, but not its `inventory` count.
3. Create a `new` product page so you can add new products — otherwise the rest of this app might be kind of underwhelming!
"""